frontend/src/metabase/home/components/Onboarding/Onboarding.tsx

```diff
@@ -409,17 +409,17 @@ export const Onboarding = () => {
                   <Text>
                     {t`You can:`}
                     <ul className={S.list}>
-                      <li>{jt`organize questions into a ${
+                      <li>{jt`Organize questions into a ${
                         showMetabaseLinks ? (
                           <ExternalLink href={dashboardTabsDocsLink} key="tabs">
                             {t`dashboard with tabs`}
                           </ExternalLink>
                         ) : (
                           t`dashboard with tabs`
                         )
-                      } and add text cards.`}</li>
-                      <li>{jt`add ${(<b key="filters">{t`filters`}</b>)} to dashboards and connect them to fields on questions to narrow the results.`}</li>
-                      <li>{t`drill-through your dashboard and charts to see more detailed data underneath.`}</li>
+                      } and add text, link, and iframe cards.`}</li>
+                      <li>{jt`Add ${(<b key="filters">{t`filters`}</b>)} to dashboards and connect them to fields on questions to narrow the results.`}</li>
+                      <li>{t`Drill-through charts on your dashboard, or set up custom destinations on click.`}</li>
```

Comment:
Can we leave a variation of the old text here? not sure what the custom destinations are? I felt that people might need to understand what is drill through, therefore "to see more detailed data underneath" was there.